README.md

```diff
@@ -69,6 +69,8 @@ the appropriate `dev/.env.docker-compose*` file as a baseline for overrides.
 ### Initial Setup
 tox is used to execute all tests.
 tox is installed automatically with the `dev` package extra.
+To install tox's dependencies into your environment, run `pip install -e .[test]`.
```

Comment:
> To install tox's dependencies

Strictly speaking, installing the `[test]` extra will only install the dependencies for the `test` Tox environment. Tox has other environments which are part of the development cycle, including `lint` and `format`.

Reply:
```suggestion
To install the tox pytest dependencies into your environment, run `pip install -e .[test]`.
```